.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This looks like user preferences, It's not a meaningful configuration, therefore this file is not welcome here.